bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxBindingConstants.java

```diff
@@ -39,11 +39,30 @@ public class SolaxBindingConstants {
     public static final String PROPERTY_INVERTER_TYPE = "inverterType";
 
     // List of all Channel ids
+    // Single phase specific
```

Comment:
It might avoid confusion if all channel names could now be prefixed by "CHANNEL_".

Reply:
There are enough comments for the different groups and also there is only one property type. If you insist, I can refactor every single one of them but it's a costy work with not so meaningful result imho...

Reply:
If you are using an IDE it should not be that costly to use a refactoring function for renaming? However, I do not insist.